specification/src/main/asciidoc/platform/Profiles.adoc

```diff
@@ -204,10 +204,10 @@ associated EJB QL, which have been made optional)
 * JSON-B 1.0
 * Concurrency Utilities 1.0
 * Batch 1.0
```

Comment:
All of these technologies should be expanded to the new Jakarta naming convention...  Jakarta Enterprise Beans, Jakarta Servlet, etc